### DISCUSSION TITLE
Send BG values to Wahoo Elemnt Bolt/Roam

### DISCUSSION THREAD

**Discussion by FamouZzE** [NightscoutFoundation/xDrip#3101] [Ideas]
I found the following [Issue ](https://github.com/NightscoutFoundation/xDrip/issues/1654)and commented there, but since it is over 3 Years old, I thought it would be the right way to open a discussion.

So basically it is already possible to install the xdrip+ Micro APK onto the Wahoo Elemnt Roam/Bolt v1 and if you are a user of a Dexcom CGM you can pair with the Wahoo. Since I am using a Freestyle Libre 3 and the Nightscout Connector, I seem to be unable to get my BG Values to my Elemnt Roam. From my understanding it should not be that hard to get these Values there since Wahoo is running simply android and the App is already installed.  I tried to sort myself through the Code and would love to contribute, but I think this is currently above my abilities.

Maybe someone can help me there or we could add this feature since it would enable users of a Wahoo Element Roam/Bolt v1 to sync any xdrip source onto their device.

**Comment by lonoel** [comment]
Hmmm... I have the same question. Someone got an answer?
